src/Sylius/Bundle/ApiBundle/Command/GuestSChangePaymentMethod.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\ApiBundle\Command;
+
+/** @experimental */
+class GuestSChangePaymentMethod extends AbstractPaymentMethod
```

Comment:
I would be for using `ChangePaymentMethod` instead of creating this one, as the intention and the behaviour should be the same

Reply:
wdyt @lchrusciel ?